Applications/CLI/ogs.cpp

```diff
@@ -97,6 +101,11 @@ int main(int argc, char *argv[])
     INFO("This is OpenGeoSys-6 version %s.",
          BaseLib::BuildInfo::git_describe.c_str());
 
+// Enable floating point exceptions for debugging.
+#ifndef NDEBUG
+    feenableexcept(FE_DIVBYZERO | FE_INVALID | FE_OVERFLOW);
```

Comment:
I'd add a command line switch that enables/disables this feature, because even when debugging one might not always want the progrm to crash, e.g. when dividing by zero.